docs/book/installation/installation.rst

```diff
@@ -64,10 +64,14 @@ After everything is in place, run the following command to install Sylius:
 Configuring Mailer
 ------------------
 
-In order to send emails you need to configure Mailer Service. Basically you need to:
+In order to send emails you need to configure Mailer Service. Basically there are multiple ways to do it:
+
+* We are recommending to use `Symfony Mailer <https://symfony.com/doc/current/mailer.html>`_ where out of the box, you can deliver emails by configuring the ``MAILER_DSN`` variable in your .env file.
+* In Symfony Mailer use the `3rd Party Transports <https://symfony.com/doc/current/mailer.html#using-a-3rd-party-transport>`_
+* (deprecated) Use SwiftMailer with this short configuration:
 
 1. **Create an account on a mailing service.**
-2. **In your** ``.env`` **file modify the** ``MAILER_URL`` **variable.**
+2. **In your** ``.env`` **file modify the** ``MAILER_DSN`` **variable.**
```

Comment:
```suggestion
2. **In your** ``.env`` **file modify the** ``MAILER_URL`` **variable.**
```

This message: 
<img width="838" alt="image" src="https://user-images.githubusercontent.com/6213903/203033398-c740849d-8e18-49f7-96bd-855efa6c116b.png">

could be misleading, but I meant that this part remains as it was, but in the first dot we should mention `MAILER_DSN`